src/Groups/pcgroup.jl

```diff
@@ -923,3 +923,58 @@ function collector(::Type{T}, G::PcGroup) where T <: IntegerUnion
 end
 
 collector(G::PcGroup) = collector(ZZRingElem, G)
+
+# GAP wrappers for group encoding / decoding
+
+"""
+    code_pcgroup(G::PcGroup)
+
+Return the code (an `Int` vector) representing the polycyclic group `G`,
+using the same encoding as GAP's `CodePcGroup` and Magma's
+`SmallGroupEncoding`.
+
+# Examples
+```jldoctest
+julia> G = pc_group(small_group(12, 2))
+Pc group of order 12
+
+julia> code = code_pcgroup(G)
+266
+
+julia> H = pcgroup_code(code)
+Pc group of order 12
+
+julia> code_pcgroup(G) == code_pcgroup(H)
+true
+```
+"""
+function code_pcgroup(G::PcGroup)
+    return GAP.Globals.CodePcGroup(GapObj(G))
+end
+
+"""
+    pcgroup_code(code)
+
+Given a code (typically an integer vector), return the polycyclic
+group it encodes.  The accepted codes and resulting groups match
+those of GAP's `PcGroupCode` and Magma's `SmallGroupDecoding`.
+
+# Examples
+```jldoctest
+julia> G = pc_group(small_group(12, 2))
```

Comment:
```suggestion
julia> G = small_group(PcGroup, 12, 2)
```

Reply:
this has not yet been addressed

Reply:
bump

Reply:
Actually this should do it also and is preferable:
```suggestion
julia> G = small_group(12, 2)
```